### ISSUE TITLE
parse in path variable name with dash error

### ISSUE THREAD

**Issue by Mr-sgreen** [acacode/swagger-typescript-api#184]
Condition:

part of the swagger json file as follows:
```json
"/location/{location-code}/content": {
			"get": {
				"summary": "find location content by location code",
				"deprecated": false,
				"produces": [
					"*/*"
				],
				"operationId": "findLocationPushContentUsingGET",
				"responses": {
					"200": {
						"schema": {
							"type": "array",
							"items": {
								"$ref": "#/definitions/PushContentVo"
							}
						},
						"description": "OK"
					},
					"401": {
						"description": "Unauthorized"
					},
					"403": {
						"description": "Forbidden"
					},
					"404": {
						"description": "Not Found"
					}
				},
				"parameters": [
					{
						"in": "path",
						"name": "location-code",
						"description": "location-code",
						"type": "string",
						"required": true
					},
				],
				"tags": [
					"push-location-controller"
				]
			}
}
```

Behavior:
throw error that location-code is illegal variable name in function parameter define
```typescript
 SyntaxError: ')' expected. (2124:48)
2122 |  * @request GET:/location/{location-code}/content
  2123 |  */
> 2124 | findLocationPushContentUsingGet: (location-code: string, params: RequestParams = {}) =>
       |                                                ^
  2125 |     this.request<(PushContentVo)[], void>({
  2126 |         path: `/location/${location-code}/content`,
  2127 |         method: 'GET',
```

Expect: 
swagger-typescript-api to auto filter dash and change to lowerCamelCase, in this case should be locationCode

**Comment by js2me** [comment]
Hello @Mr-sgreen !  
Thanks for this issue, will fix that in an hour

**Comment by Mr-sgreen** [comment]
Thank you！

> Hello @Mr-sgreen !
> Thanks for this issue, will fix that in an hour

**Comment by js2me** [comment]
@Mr-sgreen should be fixed in 6.1.0 release!  
Thank you!
